### ISSUE TITLE
Question about ckpt and training detail

### ISSUE THREAD

**Issue by Rayn-Wu** [VISION-SJTU/VEON#1]
Hi, thanks for sharing this great project!

I’m wondering if it would be possible to release the Stage 1 fine-tuned DepthAnythingV2 weights.
Additionally, I have a couple of questions:

1. If I use a depth prediction network similar to BEVDepth instead of a depth foundation model, will the performance drop significantly?

2. If I want to train the model on a 4090 GPU (24GB), would the memory be sufficient?

**Comment by zhengjilai** [comment]
1. According to the cooperation protocol with the company, we have no access to the pretrained checkpoints outside the company. But you can train one according to the tutorial.  
2. It is difficult to train a depth module simply from the CLIP features. The depth precision will be poor. But a precise off-the-shelf metric depth estimation module will be compatible with our decoupled pipeline. 
3. By default, we use 8 NVIDIA V100 GPUs with 32G memory each. It seems that 25-30G memory on each GPU is necessary.

**Comment by Rayn-Wu** [comment]
Thanks for your reply!
